### ISSUE TITLE
Error after upgrading to Atom 1.9.0 beta

### ISSUE THREAD

**Issue by NeedsAdjustment** [shd101wyy/markdown-preview-enhanced#53]
**Atom Version**: 1.9.0-beta0
**System**: Microsoft Windows 10 Home Insider Preview
**Thrown From**: [markdown-preview-enhanced](https://github.com/shd101wyy/markdown-preview-enhanced) package, v0.7.0

### Stack Trace

Uncaught TypeError: Cannot read property 'getPaths' of undefined

```
At C:\Users\joshu\.atom\packages\markdown-preview-enhanced\lib\md-preview.js:63

TypeError: Cannot read property 'getPaths' of undefined
    at MarkdownPreview.bindEditor (C:\Users\joshu\.atom\packages\markdown-preview-enhanced\lib\md-preview.js:63:47)
    at startMDPreview (C:\Users\joshu\.atom\packages\markdown-preview-enhanced\lib\md-preview.js:491:27)
    at atom-workspace.toggleMarkdownPreviewEnhanced (C:\Users\joshu\.atom\packages\markdown-preview-enhanced\lib\markdown-preview-enhanced.js:21:5)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\joshu\AppData\Local\atom\app-1.9.0-beta0\resources\app.asar\src\command-registry.js:260:29)
    at C:\Users\joshu\AppData\Local\atom\app-1.9.0-beta0\resources\app.asar\src\command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (C:\Users\joshu\AppData\Local\atom\app-1.9.0-beta0\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (C:\Users\joshu\AppData\Local\atom\app-1.9.0-beta0\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:388:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (C:\Users\joshu\AppData\Local\atom\app-1.9.0-beta0\resources\app.asar\src\window-event-handler.js:98:36)
    at HTMLDocument.<anonymous> (C:\Users\joshu\AppData\Local\atom\app-1.9.0-beta0\resources\app.asar\src\window-event-handler.js:3:61)
```

### Commands

```
     -8:00.5.0 markdown-writer:insert-new-line (atom-text-editor.editor.is-focused)
     -8:00.5.0 editor:newline (atom-text-editor.editor.is-focused)
 10x -7:58.6.0 core:backspace (atom-text-editor.editor.is-focused)
     -7:35.5.0 settings-view:open (atom-text-editor.editor)
     -7:26.1.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -6:56 markdown-preview-plus:toggle (atom-text-editor.editor.is-focused)
     -6:43.3.0 core:move-right (atom-text-editor.editor.is-focused)
     -6:43 core:backspace (atom-text-editor.editor.is-focused)
  2x -6:12.9.0 markdown-writer:insert-new-line (atom-text-editor.editor.is-focused)
     -6:12.7.0 editor:newline (atom-text-editor.editor.is-focused.autocomplete-active)
     -6:11.7.0 core:paste (atom-text-editor.editor.is-focused)
     -6:02.5.0 core:backspace (atom-text-editor.editor.is-focused)
     -5:19.7.0 settings-view:open (atom-text-editor.editor)
     -2:43.9.0 markdown-preview-enhanced:toggle (atom-text-editor.editor.is-focused)
  2x -2:21.1.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:04.2.0 markdown-preview-enhanced:toggle (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "autoHideMenuBar": true,
    "disabledPackages": [
      "gpp-compiler",
      "terminal-plus",
      "language-gfm",
      "markdown-preview",
      "fuzzy-theme-switcher",
      "theme-switch",
      "activate-power-mode",
      "atom-bitcoin",
      "fonts",
      "markdown-preview-plus",
      "markdown-scroll-sync"
    ],
    "packagesWithKeymapsDisabled": [],
    "projectHome": "F:\\OneDrive",
    "themes": [
      "atom-material-ui",
      "gruvbox-plus-syntax"
    ]
  },
  "markdown-preview-enhanced": {
    "enableWikiLinkSyntax": false,
    "exportPDFPageFormat": "A4",
    "marginsType": "minimum margin",
    "useGitHubStyle": false,
    "useGitHubSyntaxTheme": false
  }
}
```

### Installed Packages

```coffee
# User
activate-power-mode, v0.5.2 (inactive)
atom-bitcoin, v0.6.0 (inactive)
atom-material-ui, v1.2.13 (active)
atom-ternjs, v0.14.1 (active)
autocomplete-paths, v1.0.2 (active)
bitcoin-status-bar, v0.2.0 (active)
file-icons, v1.7.14 (active)
fonts, v1.5.0 (inactive)
gcc-make-run, v0.2.8 (active)
gpp-compiler, v2.3.1 (inactive)
gruvbox-plus-syntax, v1.2.2 (active)
language-matlab, v0.2.1 (active)
language-pfm, v0.82.0 (active)
linter, v1.11.8 (active)
linter-alex, v3.1.0 (active)
markdown-preview-enhanced, v0.7.0 (active)
markdown-preview-plus, v2.4.0 (inactive)
markdown-scroll-sync, v2.1.2 (inactive)
markdown-writer, v2.3.4 (active)
minimap, v4.24.4 (active)
pdf-view, v0.50.0 (active)
pen-paper-coffee-syntax, v0.14.0 (inactive)
pigments, v0.29.2 (active)
platformio-ide-terminal, v2.0.10 (active)
preview-inline, v1.0.4 (active)
script, v3.7.3 (inactive)
swackets, v0.10.0 (active)
theme-switcher, v1.1.0 (active)
tool-bar, v0.4.0 (active)
tool-bar-almighty, v0.6.1 (active)
wordcount, v2.6.0 (active)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.51.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.43.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.3.2 (inactive)
one-light-ui, v1.3.2 (inactive)
one-dark-syntax, v1.2.0 (inactive)
one-light-syntax, v1.2.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.2 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.1 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.0 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.0 (active)
bookmarks, v0.41.0 (active)
bracket-matcher, v0.82.1 (active)
command-palette, v0.38.0 (active)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.38.1 (active)
fuzzy-finder, v1.2.0 (active)
git-diff, v1.0.1 (active)
find-and-replace, v0.198.0 (active)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.1 (active)
image-view, v0.58.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (inactive)
metrics, v0.53.1 (active)
notifications, v0.64.1 (active)
open-on-github, v1.1.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.238.0 (active)
snippets, v1.0.2 (active)
spell-check, v0.67.1 (active)
status-bar, v1.3.1 (active)
styleguide, v0.46.0 (active)
symbols-view, v0.113.0 (inactive)
tabs, v0.98.0 (active)
timecop, v0.33.1 (active)
tree-view, v0.208.0 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.34.0 (active)
whitespace, v0.32.2 (active)
wrap-guide, v0.38.1 (active)
language-c, v0.52.1 (active)
language-clojure, v0.21.0 (active)
language-coffee-script, v0.47.0 (active)
language-csharp, v0.12.1 (active)
language-css, v0.36.2 (active)
language-gfm, v0.86.0 (inactive)
language-git, v0.13.0 (active)
language-go, v0.42.0 (active)
language-html, v0.44.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.22.0 (active)
language-javascript, v0.119.0 (active)
language-json, v0.18.0 (active)
language-less, v0.29.3 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.35.0 (active)
language-php, v0.37.0 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.0 (active)
language-ruby, v0.68.5 (active)
language-ruby-on-rails, v0.25.0 (active)
language-sass, v0.52.0 (active)
language-shellscript, v0.22.3 (active)
language-source, v0.9.0 (active)
language-sql, v0.21.1 (active)
language-text, v0.7.1 (active)
language-todo, v0.28.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.8 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```

**Comment by shd101wyy** [comment]
Hi, do you still get the issue after you restart atom?

**Comment by NeedsAdjustment** [comment]
Yes. Also after uninstalling/reinstalling the package. Markdown-preview-plus and markdown-preview work fine.

**Comment by xu4wang** [comment]
does it related to the project folder?
If you have the project folder added. The issue will not be triggered?

**Comment by shd101wyy** [comment]
I think that is because the beta version of **atom** has different API from current stable version.

**Comment by shd101wyy** [comment]
Hi @NeedsAdjustment , should be fixed now

**Comment by NeedsAdjustment** [comment]
Thanks! In the next update?

**Comment by shd101wyy** [comment]
@NeedsAdjustment I just published a new version `0.7.1`

**Comment by NeedsAdjustment** [comment]
Works great. Thanks again!
